src/utils/getArticleContents.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const fluid = require("infusion");
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+/*
+ * Given page content markup, this function prepares a list of links to all headings of a given level
+ * followed by the page content with ID's injected for each linked heading.
+ *
+ * This work was undertaken to address GitHub issue #59:
+ * https://github.com/fluid-project/handbook.floeproject.org/issues/59
+ *
+ * @param {String} pageContent - the page content markup
+ * @param {String} summary - the summary text to display at the top of the article contents list
+ * @param {String} [headingsSelector] - a CSS selector to indicate which heading elements to list
+ *
+ * @return {String} - the combined string containing the list markup and modified page markup
+ */
+module.exports = (pageContent, summary, headingsSelector) => {
+    const dom = new JSDOM(pageContent);
+    const headings = dom.window.document.querySelectorAll(headingsSelector || "h2");
```

Comment:
you can specify a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) `(pageContent, summary, headingsSelector = "h2")`. Instead of the guard in querySelectorAll arguments.